### PR TITLE
Update docker dependency

### PR DESCRIPTION
**Pull request by benmoss**
Part 1 of fixing buildpack/pack#102. Need to update the pack CLI as well.

### CODE REVIEW

**Comment by benmoss** [comment]
https://github.com/moby/moby/pull/38398

File: go.mod [hunk 1]
```diff
@@ -15,6 +14,7 @@ require (
 	github.com/google/go-containerregistry v0.0.0-20190110221225-f514e780f7cd
 	github.com/gorilla/context v1.1.1 // indirect
 	github.com/gorilla/mux v1.6.2 // indirect
+	github.com/morikuni/aec v0.0.0-20170113033406-39771216ff4c // indirect
```
Comment (ashwin-venkatesh) [inline go.mod]: Is this the new library that is used by docker?

Comment (benmoss) [reply go.mod]: yup, it's mentioned in https://github.com/moby/moby/pull/38398

**Comment by sclevine** [comment]
Thanks for the contribution!

**Pull request merged**